src/Orchard.Web/Core/Reports/Views/Admin/Display.cshtml

```diff
@@ -3,37 +3,34 @@
 
 @{ Layout.Title = T("Display Report").ToString(); }
 
-@using(Html.BeginFormAntiForgeryPost()) {
-    @Html.ValidationSummary()
-    <fieldset>
-        <table class="items" summary="@T("This is a table of the reports in your application")">
-            <colgroup>
-                <col id="Col1" />
-                <col id="Col2" />
-                <col id="Col3" />
-                <col id="Col4" />
-            </colgroup>
-            <thead>
-                <tr>
-                    <th scope="col">@T("Type")</th>
-                    <th scope="col">@T("Message")</th>
-                    <th scope="col">@T("Date")</th>
-                    <th scope="col"></th>
-                </tr>
-            </thead>
-            @foreach (var reportEntry in Model.Report.Entries) {
+
+<fieldset>
+    <table class="items" summary="@T("This is a table of the reports in your application")">
+        <colgroup>
+            <col id="Col1" />
+            <col id="Col2" />
+            <col id="Col3" />
+        </colgroup>
+        <thead>
+            <tr>
+                <th scope="col">@T("Type")</th>
+                <th scope="col">@T("Message")</th>
+                <th scope="col">@T("Date")</th>
+            </tr>
+        </thead>
+        @foreach (var reportEntry in Model.Report.Entries)
+        {
```

Comment:
Please format your code so that curly braces start at the end of the line instead of on a new line.